docs/user-guide/kubectl-conventions.md

```diff
@@ -29,6 +29,40 @@ In order for `kubectl run` to satisfy infrastructure as code:
 * If features are needed that are not expressible via `kubectl run` flags, switch to configuration files checked into source control.
 * Pin to a specific generator version, such as `kubectl run --generator=deployment/v1beta1`
 
+#### Generators
+
+`kubectl run` allows you to generate following resources (using `--generator` flag):
```

Comment:
OMG, that *always* bites me :wink: I need to write it down with big letters to remember for ever.